src/plugins/console/public/application/containers/editor/editor.tsx

```diff
@@ -71,9 +71,12 @@ export const Editor = memo(
     const [fetchingMappings, setFetchingMappings] = useState(false);
 
     useEffect(() => {
-      const subscription = getAutocompleteInfo().mapping.isLoading$.subscribe(setFetchingMappings);
```

Comment:
nit: we could rename `setFetchingMappings` (and `fetchingMappings`) to `setFetchingAutocompleteEntities` or something similar